src/libraries/StackUint256.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.4.13;
+
+import "ROOT/libraries/DelegationTarget.sol";
+import "ROOT/legacy_reputation/Ownable.sol";
+
+contract StackUint256 is DelegationTarget, Ownable {
+    
+    uint256[] public collection;
+    uint256 public head;
+    address public owner;
+    bool public initialized;
```

Comment:
I think we decided to make all member variables private by default, so the visibility of these will probably need to be private, and we may need getter functions for these as well.